dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1008,7 +1008,7 @@ javaCmd()
     "${JAVA_CMD}" "$@" >> "${JAVA_CMD_LOG}" 2>&1 &
   elif [ $ACTION = "run" ] || [ $ACTION = "debug" ]; then
     exec "${JAVA_CMD}" "$@"
-  elif [ $ACTION = "checkpoint" ]; then
+  elif [ $ACTION = "checkpoint" ] && [ -d "${SERVER_OUTPUT_DIR}/server.xml" ]; then
```

Comment:
`SERVER_CONFIG_DIR` should be used instead of `SERVER_OUTPUT_DIR` here.  The `SERVER_OUTPUT_DIR` defaults to the same spot as `SERVER_CONFIG_DIR` but it doesn't have to.